test/unit/test_randomai.gd

```diff
@@ -560,3 +560,6 @@ func test_chaos_100():
 
 func test_tinker_100():
 	run_iterations_with_deck("tinker")
+	
+func test_cviper_1():
```

Comment:
can you name it 100 like the rest?
Assuming you've actually run it with RandomIterations set to 100 at the top of the file, and also ran all the tests once, then we're good.

Reply:
how do i run it exactly after i set up everything?